packages/styles/extends.import.styl

```diff
@@ -0,0 +1,15 @@
+.header-font
+  font-family 'Oswald', sans-serif
+
+.body-font
+  font-family 'Open Sans', sans-serif
+
+.icon
+  font-family FontAwesome
+  -webkit-font-smoothing antialiased
```

Comment:
I think font-smoothing is only available in webkit bowsers and firefox. I'll try changing it to `font-smoothing` and see if nib adds the prefixes.